docs/getting_started.rst

```diff
@@ -6,59 +6,94 @@
 Getting Started
 """""""""""""""
 
-Using Unidist
-=============
+unidist provides :doc:`a high-level API</flow/unidist/api>` to make distributed applications. To tune
+unidist's behavior user has several methods described in :doc:`unidist configuration setting</flow/unidist/config>`
+section.
+
+Using unidist API
+=================
+
+A short example below describes how to use unidist API to organize a parallel execution for
+functions (tasks) and classes (actors).
 
 .. code-block:: python
 
+   # script.py
    if __name__ == "__main__":
+      import unidist.config as cfg
       import unidist
 
       # Initialize unidist's backend. The Ray backend is used by default.
       unidist.init()
 
-      # Apply decorator to make `f` remote function.
+      # Apply decorator to make `square` remote function.
       @unidist.remote
-      def f(x):
+      def square(x):
          return x * x
 
       # Asynchronously execute remote function.
-      refs = [f.remote(i) for i in range(4)]
-
-      # Get materialized data.
-      print(unidist.get(refs)) # [0, 1, 4, 9]
+      square_refs = [square.remote(i) for i in range(4)]
```

Comment:
Where is getting results?

Reply:
This is placed after cubes computing. There is no `get` here to highlight that references could be provided in the next remote function without forcing computing using `get`.

Reply:
I do not see getting materialized data on `square_refs`.

Reply:
I'm talking about this above. There is `get` only for `cubes_refs`.

Reply:
This is strange to me that we only show materializing data `cubes_refs`. Let's add it for `square_refs` too.

Reply:
In this case it may gives an impression, that need to call `get` after the each remote call. Using `get` in the end of workflow is a better practice to not ruin async execution. I still prefer to leave it as it is.

@no-ponomarev, what's your opinion?

Reply:
We can add it nearby getting materialized data for `cude_refs`.

Reply:
What's point to see a materialized data in `square_refs` (this doesn't workflow result)?

Reply:
That example doesn't seem to me as a complete workflow. We just show how to use tasks and actors to run computation.

Reply:
I think, @YarShev missed the chaining example in this section, `square_refs` are materialized inside an actor. I missed it too firstly. And that is what we should emphasize in comments.

Reply:
No, It is intuitive that `square_refs` are materialized inside an actor and it is redundant to emphasize the materialization there in comments.

Reply:
It was not for me. Than why did you want to call `get` on `square_refs` if they are already materialized through chaining?

Reply:
Those materialized in a worker but not in the main process.

Reply:
I added `unidist.get(square_refs)` call before `unidist.get(cude_refs)`. I think @YarShev only wants to demonstrate, additionally, what data are placed behind `square_refs` .

Reply:
Maybe just to add comment that you can call `get` on square_refs but also to implement chaining?

Reply:
Chaining is shown for Cube, getting the data further. I think it is okay.